src/components/CheckBox/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react';
+
+import { StyledTodoText, StyledTodoCheckbox } from '@components/CheckBox/CheckBox.styles';
+
+type CheckBoxProps = Omit<React.InputHTMLAttributes<HTMLInputElement>, 'onChange'> & {
+  id: string;
+  labelText: string;
+  checked: boolean;
+  onChange: (id: string) => void;
```

Comment:
this shouldn't accept an `id` anymore, instead it should get the new checked value